content/300-accelerate/600-faq.mdx

```diff
@@ -8,6 +8,18 @@ toc: true
 
 <TopBlock>Below are frequently asked questions about Accelerate.</TopBlock>
 
+## When do I have to enable static IP for Prisma Accelerate?
+
+A static IP address is an IPv4 or an IPv6 address that doesn’t change. Unlike dynamic IP addresses, which can change unpredictably, traffic from static IP addresses are predictable.
```

Comment:
Let's add why this is good: allows you to know where traffic is coming from and that it's trusted, so you can let it through based on your security rules. (Please phrase it better than this :) )